api/client/network.go

```diff
@@ -222,7 +224,7 @@ func consolidateIpam(subnets, ranges, gateways []string, auxaddrs map[string]str
 				return nil, fmt.Errorf("multiple overlapping subnet configuration is not supported")
 			}
 		}
-		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}}
+		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}, Options: options}
```

Comment:
@mavenugo Well with libnetwork it has its ipam options defined for each subnet and I was just following that structure as seen here: https://github.com/docker/libnetwork/blob/master/docs/ipam.md#ipam-configuration-and-flow

Even though it is generic options it seems like libnetwork still includes it for each subnet. I could be interpreting it wrong but that was just my understanding.